### PR TITLE
Use env vars instead of positional args for stage/desired_version

### PR DESCRIPTION
**Pull request by vinnyjth**
Changed the positional args on the deploy scripts to env args so they could be controlled easier on bitrise. 

## 🔬 To Test
1. `./deploy_android`
2.

## 📸 Screenshots
Was able to build fellowship greenville in Bitrise with an initial version of 1.0.0

CleanShot 2023-01-30 at 16 27 33

## 📝 Checklist

- [ ] I have performed a self-review of my code
- [ ] I have tested my code works in the project (nothing breaks)
- [ ] I have commented my code in hard-to-understand areas
- [ ] New and existing tests and linting pass locally with my changes
- [ ] I have made corresponding changes to the documentation (new envs?)
- [ ] For PR's with design updates/changes, I have consulted with the product designer and made all necessary adjustments in Figma

### CODE REVIEW

File: .github/workflows/deploy-mobile.yml [hunk 1]
```diff
@@ -37,10 +37,12 @@ jobs:
         env:
           APOLLOS_API_KEY: ${{ secrets.APOLLOS_API_KEY }}
           CHURCH: ${{ github.event.inputs.church }}
-      - run: scripts/deploy-ios.sh internal ${{ github.event.inputs.iosVersion }}
+      - run: scripts/deploy-ios.sh
         env:
           APOLLOS_API_KEY: ${{ secrets.APOLLOS_API_KEY }}
           CHURCH: ${{ github.event.inputs.church }}
+          STAGE: internal
```
Comment (redreceipt) [inline .github/workflows/deploy-mobile.yml:44]:
```suggestion
          TRACK: internal
```

File: .github/workflows/deploy-mobile.yml [hunk 2]
```diff
@@ -58,7 +60,9 @@ jobs:
         env:
           APOLLOS_API_KEY: ${{ secrets.APOLLOS_API_KEY }}
           CHURCH: ${{ github.event.inputs.church }}
-      - run: scripts/deploy-android.sh internal ${{ github.event.inputs.androidVersion }}
+      - run: scripts/deploy-android.sh
         env:
           APOLLOS_API_KEY: ${{ secrets.APOLLOS_API_KEY }}
           CHURCH: ${{ github.event.inputs.church }}
+          STAGE: internal
```
Comment (redreceipt) [inline .github/workflows/deploy-mobile.yml:67]:
```suggestion
          TRACK: internal
```

File: .github/workflows/deploy-mobile.yml [hunk 3]
```diff
@@ -58,7 +60,9 @@ jobs:
...
+          DESIRED_VERSION: ${{ github.event.inputs.iosVersion }}
```
Comment (redreceipt) [inline .github/workflows/deploy-mobile.yml:68]:
```suggestion
          DESIRED_VERSION: ${{ github.event.inputs.androidVersion }}          
```

File: .github/workflows/deploy-mobile.yml (hunk 1 above)
Comment (vinnyjth) [reply .github/workflows/deploy-mobile.yml:44]: thanks ❤️

**Review by redreceipt — CHANGES_REQUESTED** [review]
left some comments

**Review by redreceipt — APPROVED** [review]

**Pull request merged**